tasks/new_e2e_tests.py

```diff
@@ -119,6 +120,7 @@ def run(
         "src_agent_version": f"-src-agent-version {src_agent_version}" if src_agent_version else '',
         "dest_agent_version": f"-dest-agent-version {dest_agent_version}" if dest_agent_version else '',
         "keep_stacks": '-keep-stacks' if keep_stacks else '',
+        "replace_stacks": '-replace-stacks' if replace_stacks else '',
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I would rather have an `extra_flags` to pass custom flags, `replace_stacks` should not be exposed here as it is defined only in your test

Reply:
sounds good
8e2907a21481e30bf8974efe15a4a90ccc3b34e5